backend/migrations/versions/72b8a90b6ee8__share_request_purpose.py

```diff
@@ -0,0 +1,136 @@
+"""_share_request_purpose
+
+Revision ID: 72b8a90b6ee8
+Revises: 509997f0a51e
+Create Date: 2023-06-05 12:28:56.221364
+
+"""
+from alembic import op
+from sqlalchemy import orm, Column, String, and_
+from sqlalchemy.ext.declarative import declarative_base
+
+from dataall.db import api, models, permissions
```

Comment:
"old" imports have to be updated

Reply:
Yes, good call! 
I will update all review remarks in one bunch once you are done :)